src/kanata/mod.rs

```diff
@@ -5,10 +5,14 @@ use log::{error, info};
 use parking_lot::Mutex;
 use std::sync::mpsc::{Receiver, SyncSender as Sender, TryRecvError};
 
+#[cfg(feature = "passthru_ahk")]
+use std::sync::mpsc::Sender as ASender;
```

Comment:
I think this would look nicer moved down to line 445

Reply:
it would, but it's part of a function signature, so need to import before, and can't import in Impls